Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -1068,6 +1068,89 @@
   - type: Sprite
     state: poster52_legit
+    
+# Goob posters
+
+- type: entity
+  parent: PosterBase
+  id: PosterLegitBigBrotherAI
+  name: "AI Sees All"
+  description: "A poster that reminds you that your secrets are not hidden from your Station AI"
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)